index.js

```diff
@@ -0,0 +1,14 @@
+class Dropdown {
```

Comment:
One thing I would say that you can always think about is that when I hover over the dropdown menu, the mouse does not change to a pointer. But this was not asked and is just a small suggestion!